Added documentation policy to README.md (#834)
microsoft/terminal: README.md

> ⚠ **Note**: The Command-Line Team is actively working out of this repository and will be periodically re-structuring the code to make it easier to comprehend, navigate, build, test, and contribute to, so **DO expect significant changes to code layout on a regular basis**.

## Documentation

> All documentation is located in the ./docs folder. If you would like to contribute to the documentation, please complete a pull request with any new changes.

Reviewer: ```suggestion
All documentation is located in the `./docs` folder. If you would like to contribute to the documentation, please submit a pull request.
```
